libpromises/evalfunction.c

```diff
@@ -8601,6 +8666,16 @@ static const FnCallArg READFILE_ARGS[] =
     {NULL, CF_DATA_TYPE_NONE, NULL}
 };
 
+static const FnCallArg CLASSEXPRESSION_FILTERDATA_ARGS[] =
+{
+    {CF_ABSPATHRANGE, CF_DATA_TYPE_STRING, "File name to read"},
+    {CF_VALRANGE, CF_DATA_TYPE_INT, "Column with class expression"},
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "String to delimit data by"},
+    {CF_BOOL, CF_DATA_TYPE_OPTION, "CSV file has heading"},
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "Column to sort by"},
```

Comment:
Is this the column name or column index position?
If it is an index position, is it counting from 0 or 1?
If it is can be a column name, will it error if the column name is not found?

Reply:
Currently not implemented properly. I think that this should be explicitly index, but in case you want both to be possible, it should be decided based on the `has_heading` parameter.

Reply:
Add the word index to description.